instrumentation/jdbi3/src/test/java/brave/jdbi3/Jdbi3BravePluginIT.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package brave.jdbi3;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.core.JdbiException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import brave.Span;
+import brave.Tracing;
+import brave.handler.MutableSpan;
+import brave.handler.SpanHandler;
+import brave.propagation.TraceContext;
+
+class Jdbi3BravePluginIT {
```

Comment:
try for naming conventions and other conventions in the project. IT prefix e.g. https://github.com/openzipkin/brave/blob/master/instrumentation/mysql8/src/test/java/brave/mysql8/ITTracingQueryInterceptor.java and also IT implies real DB. This might be a unit test? ack jdbc:h2:mem is grey area.

Reply:
It´s the use of an actual (though in-process and in-memory) database, along with using the third-party libraries as-is (as opposed to stubbing them, or parts of them, out) that made me call it an IT, but if you disagree, I´ll gladly change it.

Reply:
I´ve changed the tests over to more-or-less a copy of the tests from the mysql8 instrumentation. I´ll make a stab at getting the tests run with both H2 and MySQL, as a way to show that the instrumentation is not tied to any particular database.

Reply:
Made the test abstract, with concrete implementations that test against H2 in-memory, MySQL configured as in the mysql8 tests, Postgres via the Zonky.io extension, and Sqlite via the xerial.org extension.
In Junit 5.13, there will be a cleaner way of doing this, but it´s not released yet.